<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab2d</class>
 <widget class="QWidget" name="tab2d">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>475</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>File preview:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="twod_previewedit">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Datasets:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryCombo" name="twod_datasetsedit">
       <property name="toolTip">
        <string>A space separated list of dataset names to import from the file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="twod_helpbutton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Range of &amp;X:</string>
       </property>
       <property name="buddy">
        <cstring>twod_xminedit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryCombo" name="twod_xminedit">
       <property name="toolTip">
        <string>Enter a number for the minimum coordinate of the X axis (default 0)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryCombo" name="twod_xmaxedit">
       <property name="toolTip">
        <string>Enter a value for a maximum coordinate of the X axis (default is number of columns in file)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Range of &amp;Y:</string>
       </property>
       <property name="buddy">
        <cstring>twod_yminedit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryCombo" name="twod_yminedit">
       <property name="toolTip">
        <string>Enter a number for the minimum coordinate of the Y axis (default 0)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryCombo" name="twod_ymaxedit">
       <property name="toolTip">
        <string>Enter a number for the maximum coordinate of the Y axis (default number of rows in file)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="HistoryCheck" name="twod_invertrowscheck">
       <property name="text">
        <string>Invert &amp;rows</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="HistoryCheck" name="twod_transposecheck">
       <property name="toolTip">
        <string>Transpose rows and columns</string>
       </property>
       <property name="text">
        <string>Trans&amp;pose</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="HistoryCheck" name="twod_invertcolscheck">
       <property name="text">
        <string>Invert &amp;columns</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="HistoryCheck" name="twod_gridatedge">
       <property name="toolTip">
        <string>The centres of the pixels are given in the top row and the left column</string>
       </property>
       <property name="text">
        <string>Grid at edge</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryCombo</class>
   <extends>QComboBox</extends>
   <header>historycombo.h</header>
  </customwidget>
  <customwidget>
   <class>HistoryCheck</class>
   <extends>QCheckBox</extends>
   <header>historycheck.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
